cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCoapStackFactory.java

```diff
@@ -27,8 +27,8 @@
 import org.eclipse.californium.elements.config.Configuration;
```

Comment:
Please remove the

```
import org.eclipse.californium.core.network.CoapStackFactory;
```

above. It's not longer required and reported with a "deprecated warning".

Reply:
Done in commit 2861fb22ba8d37687159909b017663b15631fded